README.md

```diff
@@ -60,18 +60,36 @@ cannot open 'zroot/iocell/jails/XXXXXXXX-XXXX-XXXX-a5be-3417ebb13845': dataset d
 
 **QUICK HOWTO:**
 
-Fetch a release:
+`iocell(8)` requires a zfs pool to store its jails, templates and configuration.
+FreeBSD's default installer uses `zroot` as the default name for the system
+zpool, so these examples below do too. If you wish, you may use an alternate
+zpool, whether due to differing install-time choices, or if you have another
+zpool to use instead.
```

Comment:
Change to "whether due to different install-time choices, or if you have dedicated zpool for iocell to use."